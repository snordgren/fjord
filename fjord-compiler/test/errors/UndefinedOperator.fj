module test.errors.UndefinedOperator

undefinedOp : Int -> Int
undefinedOp a = a ++ 1
