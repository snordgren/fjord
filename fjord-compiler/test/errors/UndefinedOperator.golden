test/errors/UndefinedOperator.fj:4:18:
  |
4 | undefinedOp a = a ++ 1
  |                  ^^^^^^
"++" is undefined in scope
